src/Sylius/Bundle/ApiBundle/Exception/CartNotFoundException.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Exception;
+
+use Symfony\Component\HttpKernel\Exception\UnprocessableEntityHttpException;
+
+final class CartNotFoundException extends UnprocessableEntityHttpException
```

Comment:
```suggestion
final class UnprocessableCartException extends UnprocessableEntityHttpException
```
?
Having a `NotFound*` exception return codes other than 404 is confusing, as we already have a few actual' NotFound' exceptions in the API scope that behave as expected.